BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-1.0.25-prerelease-00182
+1.0.25-prerelease-00182
```

Comment:
NIT: whitespace change only, revert (this is probably because I updated the buildtools earlier, #867)